### DISCUSSION TITLE
设备一直重连

### DISCUSSION THREAD

**Discussion by Aquede** [immortalwrt/immortalwrt#1320] [Q&A]
我的设备是360t7，小米9，红米10a。
在无线板块将地区码改成其他地区，将无线功率改大之后。重启路由器三次或多次，我的手机就会无线重连，等待一定时间后就连上了，但是只要开关一下无线，又会无线重连，如此反复。希望能修一下。
我刷了固件有237大，h大，或者其他大佬编译的imm源码固件都重复这个问题

**Comment by xlighting2017** [comment]
意思是  无论用啥软件，都有这个问题， 那软件怎么修呢...

**Reply by Aquede** [reply]
我不知道是哪里出了问题，但我改了设置之后，重启三次及以上，小米9，红米10a就会一直断了又重连

**Reply by liyiranlab** [reply]
你无线开源闭源，我之前T7闭源wifi，一加竞速版无线重连，T7路由器5Gwifi从160改80Mhz，没有这个问题；后来改uboot 刷天灵大佬的immortalwrt 开源WiFi 开160Mhz 也没有这个问题。
https://github.com/immortalwrt/immortalwrt/discussions/1257

**Reply by Aquede** [reply]
> 你无线开源闭源，我之前T7闭源wifi，一加竞速版无线重连，T7路由器5Gwifi从160改80Mhz，没有这个问题；后来改uboot 刷天灵大佬的immortalwrt 开源WiFi 开160Mhz 也没有这个问题。 #1257

开源闭源的都刷了，我刷了潘多拉固件就不会了

**Reply by liyiranlab** [reply]
你试过immortalwrt master？我之前用h大闭源固件wifi一直重连，换uboot用immortalwrt master 就没有问题，还有判断就是关闭160Mhz。

**Reply by Aquede** [reply]
> 你试过immortalwrt master？我之前用h大闭源固件wifi一直重连，换uboot用immortalwrt master 就没有问题，还有判断就是关闭160Mhz。

有链接吗，我用别人编译的这个项目会出现我说的问题

**Reply by liyiranlab** [reply]
项目主页不就有编译好的固件，不过得自己安装luci。可以自定义打包固件。

**Comment by liyiranlab** [comment]
还有你用别人固件到这里来说是非常不礼貌的，不过广撒网容易有结果。

**Reply by Aquede** [reply]
237大，h大用的不是这个项目的源码吗？而且我刷了从其他大佬刷的固件也是这个项目的源码啊，而且标注出来了

**Reply by Aquede** [reply]
我换了很多固件，我一层层找上来的，问题是都复现了

**Reply by liyiranlab** [reply]
给你说出问题的是闭源wifi，这里是开源wifi。237大，h大用都是闭源驱动。

**Comment by igmkgu** [comment]
原来闭源也这样，我用开源偶尔也断连

**Reply by liyiranlab** [reply]
闭源以前不会断，后来不知道为什么，会好像会强制切换160Mhz，导致不支持的手机一直断；我后来用开源，开源没有断过，但是最新的版本不知道为什么，只有750兆协商速度。

**Reply by liyiranlab** [reply]
忘了说我的T7改过散热，你的断流可能是cpu 或 wifi 芯片温度过高导致。

**Reply by liyiranlab** [reply]
之前，我妈妈的一加竞速版手机，连接T7协商速度只有780兆，用ZN-M2连接协商可以1201兆;今天换了T7两个电感，2R2和1R0，更换成一体成型封装0420的优质电感。测试连接手机协商速度可以达到1201兆了。现在我不确定是更换屏蔽电感稳定了电源，使协商速度稳定，还是天气降温使手机连接稳定。但是之前相同温度环境下，M2和手机可以协商到1201兆。但现我家就一部可以上1000兆的手机，判断比较困难。等过几天天气温度上升再看看。对了，一体电感漏磁更少，对外部元件影响更小，更换后可提高电源输出质量。

**Reply by liyiranlab** [reply]
补充一下，早以前t7和一加竞速版手机的协商速度可以达到1201兆;除了改电感，在等电感的这几天，手机系统升过级。可能是手机系统手机后，协商速率才正常的。但是奇怪的是M2 路由器的速率之前正常，就T7降速。

**Comment by initzhu** [comment]
你160mhz开在5.4还是5.8Ghz？ 5.8不能开160 5.4开的话你看下频道是不是52
